doc/mediasdk-man.md

```diff
@@ -1161,6 +1165,125 @@ Assume the pipeline is **A->B->C**. The application synchronizes on sync point *
 
 ## <a id='hardware_acceleration'>Working with hardware acceleration</a>
 
+### <a id='Working_with_multiple_intel_media_devices'>Working with multiple Intel media devices</a>
+
+If your system has multiple Intel Gen Graphics adapters you may need hints on which adapter suits better to process some particular workload. The SDK provides API to help with adapter selection. See Example 9 that showcases workload processing initialization on discrete adapter.
```

Comment:
hard to read....

"Media SDK provides helper API to select best suitable adapter for your workload based on passed workload description. See Example 9 that showcases workload initialization on discrete adapter." (remove word "processing" from example line)

Reply:
ok, will do later

Reply:
done